dirsrvtests/tests/suites/password/password_TPR_policy_test.py

```diff
@@ -0,0 +1,508 @@
+
+    :expected results:
+    1. Success
+    2. Success
+    3. Fail(ldap.INSUFFICIENT_ACCESS)
+    4. Success 
+
+"""
+    log.info('Creating 2 users with appropriate password')
+    user1 = UserAccount(topo.standalone, f'uid=jdoe1,ou=People,{DEFAULT_SUFFIX}')
+    user2 = UserAccount(topo.standalone, f'uid=jdoe2,ou=People,{DEFAULT_SUFFIX}')
+
+
+    log.info('Setting Local policies...')
+    conn_user2 = user2.bind(USER2_PASS)
+    
+
+    UserAccount(conn_user2, user2.dn).replace('userpassword', 'reset_pass')  
+    log.info('Attempting to change user#1  password as user#2 ')
+
+    with pytest.raises(ldap.INSUFFICIENT_ACCESS):
+        UserAccount(conn_user2, user1.dn).replace('userpassword', 'reset_pass')
+
+def test_local_TPR_supercedes_global_TPR(topo, _add_user, set_global_TPR_policies):
+    """ One Time password with expiration
+    
+    :id: beb2dac4-e116-11eb-a85e-98fa9ba19b65
+    :setup: Standalone
+    :steps:
+    1. Create DS Instance
+    2. Create user with appropriate password
+    3. Configure the Global Password policies with passwordTPRMaxUse 5     
+    4. Configure different local password policy for passwordTPRMaxUse 3
+    5. Trigger TPR by resetting the user password above
+    6. Attempt an ldap search with an incorrect bind password for user above
+    7. Repeat as many times as set by attribute passwordTPRMaxUse
+    8. Should lock the account after value is set in the local passwordTPRMaxUse is reached
+    9. Try to search with the correct password account will be locked.
+
+    :expected results:
+    1. Success
+    2. Success
+    3. Fail(ldap.INSUFFICIENT_ACCESS)
+    4. Success
+    5. Success
+    6. Success
+    7. Success
+    8. Success
+    9. Success 
+
+"""
+
+    user1 = UserAccount(topo.standalone, f'uid=jdoe1,ou=People,{DEFAULT_SUFFIX}')
+    user2 = UserAccount(topo.standalone, f'uid=jdoe2,ou=People,{DEFAULT_SUFFIX}')
+    log.info('Setting local password Temporary password reset policies')
+     
+    log.info('Setting Global TPR policy attributes')
+    Config(topo.standalone).replace('passwordMustChange', 'on')
+    Config(topo.standalone).replace('passwordTPRMaxUse', '5')
+    Config(topo.standalone).replace('passwordTPRDelayExpireAt', '600')
+    Config(topo.standalone).replace('passwordTPRDelayValidFrom', '6')
+    log.info('Resetting {} password to trigger TPR policy'.format(user1))
+    user1.replace('userpassword', 'not_allowed_change')
+    count = 0
+    
+    while count < 4:
+        if count == 4:
+            with pytest.raises(ldap.CONSTRAINT_VIOLATION):
+                user2.bind('badbadbad')
+        else:        
+           with pytest.raises(ldap.INVALID_CREDENTIALS):
+               count+=1
+               user2.bind('badbadbad')
+
+
+def test_once_TPR_reset_old_passwd_invalid(topo, _add_user,set_global_TPR_policies):
+    """ Verify that once a password has been reset it cannot be reused
+    
+    :id: f3ea4f00-e89c-11eb-b81d-98fa9ba19b65
+    :setup: Standalone
```

Comment:
Please add a **customerscenario: True** to all the docstrings, because this is a customer case.
Here is an [example](https://github.com/389ds/389-ds-base/blob/master/dirsrvtests/tests/suites/basic/basic_test.py#L1376).

Reply:
Thank you! Will fix

Reply:
Thanks for the changes.